packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -23,6 +22,15 @@ const FeedbackStore = types
       return flatten(Array.from(self.rules.values()))
         .some(rule => rule.hideSubjectViewer)
     },
+    get isActive () {
+      const { projects = {}, subjects = {}, workflows = {} } = getRoot(self)
+      return helpers.isFeedbackActive(projects.active, subjects.active, workflows.active)
```

Comment:
What happens if these references are not valid? Should `isValidReference` be used first before calling `isFeedbackActive`?

Reply:
The function returns false if they aren't valid, I think. I may be misunderstanding what a valid reference is.

Reply:
Several of the errors I fixed in the MST upgrade had to do with attempting to use active references when they weren't valid. For safety reasons, I think it'd be best to check that they're valid first, especially if we're not sure on the behavior here.

Reply:
Done.

`isFeedbackActive()` returns false if any of its arguments are falsy but I wasn't sure if that's sufficient.